rsocket-core/src/main/java/io/rsocket/frame/FrameHeaderFlyweight.java

```diff
@@ -31,29 +12,13 @@
  *
  * <p>Not thread-safe. Assumed to be used single-threaded
  */
-public class FrameHeaderFlyweight {
-
-  private FrameHeaderFlyweight() {}
-
-  public static final int FRAME_HEADER_LENGTH;
-
-  private static final int FRAME_TYPE_BITS = 6;
-  private static final int FRAME_TYPE_SHIFT = 16 - FRAME_TYPE_BITS;
-  private static final int FRAME_FLAGS_MASK = 0b0000_0011_1111_1111;
-
+public final class FrameHeaderFlyweight {
   public static final int FRAME_LENGTH_SIZE = 3;
```

Comment:
nit: `FRAME_LENGTH_SIZE` and `FRAME_LENGTH_MASK` are TCP transport specific now, so maybe move them to `FrameLengthFlyweight`?